environment.yaml

```diff
@@ -0,0 +1,122 @@
+name: manga-image-translator
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - bzip2=1.0.8=h8ffe710_4
+  - ca-certificates=2022.12.7=h5b45459_0
+  - intel-openmp=2023.0.0=h57928b3_25922
+  - libblas=3.9.0=16_win64_mkl
+  - libcblas=3.9.0=16_win64_mkl
+  - libffi=3.4.2=h8ffe710_5
+  - libhwloc=2.9.0=h51c2c0f_0
+  - libiconv=1.17=h8ffe710_0
+  - liblapack=3.9.0=16_win64_mkl
+  - libsqlite=3.40.0=hcfcfb64_0
+  - libxml2=2.10.3=hc3477c8_6
+  - libzlib=1.2.13=hcfcfb64_4
+  - mkl=2022.1.0=h6a75c08_874
+  - openssl=3.1.0=hcfcfb64_0
+  - pip=23.0.1=pyhd8ed1ab_0
+  - pthreads-win32=2.9.1=hfa6e2cd_3
+  - pydensecrf=1.0rc3=py310h1c4a608_4
+  - python=3.10.10=h4de0772_0_cpython
+  - python_abi=3.10=3_cp310
+  - setuptools=67.6.1=pyhd8ed1ab_0
+  - tbb=2021.8.0=h91493d7_0
+  - tk=8.6.12=h8ffe710_0
+  - ucrt=10.0.22621.0=h57928b3_0
+  - vc=14.3=hb6edc58_10
+  - vs2015_runtime=14.34.31931=h4c5c07a_10

```

Comment:
I think visual studio is only needed on windows, which would confuse linux & mac users.

Reply:
To be fair who’s gonna use it on Mac or Linux. That’s a fix for people who are using windows and don’t have cython installed. When using Linux it’s installed with python. And mac doesn’t support Nvidia gpus so it’s basically unusable.( mac should install cython with python too)

Reply:
I use conda, and I use both Mac and Linux.
We already have a note just in case the compilation in Windows fails. There is no way that the wheel won't install.
Given that the issues in `pydensecrf` repo mentioned compilation in conda could also fail, and maintaining another requirement file is just an additional headache, I still think that what we should do is to just add another note saying conda may also work.

Reply:
But on Mac and Linux you can just do pip install so the file isn’t needed. The attempt with the batch file was to remove pydensecrf from the requirements and install it with conda, but it seems like it isn’t working. I don’t care either way, because i use venv and no windows. It was just a suggestion.

and when I am allowed to ask. How did you get an Nvidia gpu to run on Mac. Macs aren’t known for their high end cpus and even those struggle with machine learning.